File: src/main/java/kr/mjc/rachel/concepts/classesandobjects/Circle.java
package kr.mjc.rachel.concepts.classesandobjects;

public class Circle {
  private int x;
  private int y;
  private int radius;

  public Circle(int x, int y) {
    this.x = x;
    this.y = y;
    this.radius = 1;
  }

  public final int getX() {
    return x;
  }

  public final void setX(int x) {
    this.x = x;
  }

  public final int getY() {
    return y;
  }

  public final void setY(int y) {
    this.y = y;
  }

  public final int getRadius() {
    return radius;
  }

  public final void setRadius(int value) {
    if (value >= 0) this.radius = value;
    else throw new RuntimeException("반지름이 0 미만");
  }

  @Override
  public String toString() {
    return String.format("Circle{x=%d, y=%d, radius=%d}", x, y, radius);
  }
}

File: src/main/kotlin/kr/mjc/jacob/basicsyntax/Circle.kt
package kr.mjc.jacob.basicsyntax

const val PI = 3.14

class Circle(val x: Int, val y: Int, val radius: Int)

fun main() {
  val c1 = Circle(0, 0, 1)
  val c2 = Circle(1, 1, 10)
  println(c1.x)
  println(c2.x)
  println(PI)
}